src/olympia/addons/tests/test_models.py

```diff
@@ -1586,6 +1588,43 @@ def test_theme_is_recommended(self):
         # but not when it's removed.
         assert not addon.is_recommended
 
+    def test_promoted_group(self):
+        addon = addon_factory()
+        # default case - no group so not recommended
+        assert not addon.promoted_group
+
+        PromotedApproval.objects.create(
+            version=addon.current_version, group_id=SPOTLIGHT.id)
+        promoted = PromotedAddon.objects.create(
+            addon=addon, group_id=SPOTLIGHT.id)
+        del addon.promoted_group
```

Comment:
You need to do this because it's a `@cached_property`?

Reply:
yes.